viadot/sources/outlook.py

```diff
@@ -182,10 +182,13 @@ def _get_messages_from_mailbox(
                     recivers_list = fetched.get("toRecipients")
                     recivers = " "
                     if recivers_list is not None:
-                        recivers = ", ".join(
-                            reciver["emailAddress"]["address"]
-                            for reciver in recivers_list
-                        )
+                        for reciver in recivers_list:
+                            add_string = f", {reciver['emailAddress']['address']}"
+                            if sum(list(map(len, [recivers, add_string]))) >= 8000:
+                                break
+                            else:
+                                recivers += add_string
```

Comment:
My suggestion - what do you think about using only 1 if to this case?
```suggestion
                            if sum(list(map(len, [recivers, add_string]))) < 8000:
                               recivers += add_string
                                
```

Reply:
The en is the same, because, I need to break the `for` loop when it reaches 8000, after that limit you are iterating without any purpose.
If I use your sentence, then I would need an `else: break` which is basically the same as it already there.